includes/admin/helpers/class-sales-api.php

```
@@ -0,0 +1,154 @@
+ * @since x.x
+ */
+class WPBDP_Sales_API extends WPBDP_Modules_API {
+
+	use WPBDP_Who;
+
+	/**
+	 * @var array|false
+	 */
+	private static $best_sale;
+
+	public function __construct( $license = null ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$license'. (Unused Code Rules)

(UnusedFormalParameter)